agent/src/main/java/com/cloud/agent/AgentShell.java

```diff
@@ -539,10 +527,6 @@ public void start() {
                 s_logger.debug("[ignored] AgentShell was interrupted.");
             }
 
-        } catch (final ConfigurationException e) {
-            s_logger.error("Unable to start agent: " + e.getMessage());
-            System.out.println("Unable to start agent: " + e.getMessage());
-            System.exit(ExitStatus.Configuration.value());
         } catch (final Exception e) {
             s_logger.error("Unable to start agent: ", e);
```

Comment:
maybe get rid of the `System.out.println(..)` below as well?

Reply:
@DaanHoogland done